### PR TITLE
revert to live-serve-build

### LINKED ISSUE #277: Find a replacement for the serve-https npm package.
The [serve-https package](https://www.npmjs.com/package/serve-https) helps us run automated tests in the development environment by serving web pages over https. Usually that requires setting up certificates and a lot of other work, but the package just does everything for us. Unfortunately they do not host the package on github and their git servers keep going down, causing our automated build to fail. We should try to find alternatives to this package. We may even continue to use serve-https if they fix the problems, but having a backup plan is important so we don't slow down development when there are problems with the package.

### PR DESCRIPTION
**Pull request by tbushman**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [x] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

- [x] Closes currently open issue: Closes #277, #309

#### Description

revert to live-server

### CODE REVIEW

**Comment by tbushman** [comment]
@tchaffee what were you using for the password in `config/live-server-https` ?

**Comment by tbushman** [comment]
@tchaffee see Travis output: https://travis-ci.org/freeCodeCamp/testable-projects-fcc/builds/377567817?utm_source=github_status&utm_medium=notification) ... I think I need to spawn a separate process for serving vs running npm test ?

File: src/project-tests/pomodoro-clock-tests.js [hunk 1]
```diff
@@ -823,19 +823,18 @@ export default function createPomodoroClockTests() {
...
         assert.isTrue(
-          audioElem.paused,
+          document.getElementById('beep').paused,
           'Audio element was not stopped when reset was clicked.'
         );
 
         assert.equal(
           0,
-          audioElem.currentTime,
+          document.getElementById('beep').currentTime,
```
Comment (tbushman) [inline src/project-tests/pomodoro-clock-tests.js]: @tchaffee The pomodoro test was not passing without this change, and this PR is needed to pass subsequent PRs, so consider allowing this non-sequitur change. In my understanding, doing .getElementById each test helps bust the component's cache...

Comment (tchaffee) [reply src/project-tests/pomodoro-clock-tests.js]: I'd rather see this as a separate PR. The problem is that in the future when someone new to the project looks to reverse this PR then they see changes to Pomodoro tests and they don't know whether or not to reverse these seemingly related changes too. If you are tight on time and need help I can create a separate PR for these Pomodoro improvements.

Comment (tbushman) [reply src/project-tests/pomodoro-clock-tests.js]: OK, #312 passes local Selenium tests.

File: .travis.yml [hunk 2]
```diff
@@ -14,8 +16,11 @@ before_script:
   - "export DISPLAY=:99.0"
   - cp test/setup.js.travis test/setup.js
   # Start https server (and sleep) so we can get the local bundle.js via https.
-  - npm run serve-https-local > /dev/null 2>&1 &
-  - sleep 3
+  # This is basically the same process for generating self-signed certificates
+  # outlined in CONTRIBUTING, followed by npm run `live-serve-build &`
+  # The `&` at the end tells this process to fork into the background
+  # https://github.com/travis-ci/travis-ci/issues/1321
+  - cd config && openssl genrsa -des3 -passout pass:PASS -out server.key 2048 && openssl req -passin pass:PASS -new -key server.key -out server.csr -subj "/C=BR/ST=Parana/L=Curitiba/O=fCC/OU=test/CN=freeCodeCamp" && cp server.key server.key.org && openssl rsa -in server.key.org -passin pass:PASS -out server.key && openssl x509 -req -days 365 -in server.csr -signkey server.key -out server.crt && cp live-server-https.js.example live-server-https.js && sleep 3 && npm run live-serve-build &
```
Comment (tchaffee) [inline .travis.yml]: Thanks for this detailed documentation. Very helpful.

**Pull request merged**